.gitignore

```diff
@@ -1,5 +1,6 @@
 *.log
 npm-debug.log*
+composer.phar
```

Comment:
Not sure we need that but OK.

Reply:
What are the other options? Include it in the repo, or delete it from my machine before committing? Or, I suppose, only add it to my own `.gitignore`? This seemed like the simplest and least effort of those options, and could help others who end up running the tests locally.

Reply:
I personally have `composer` installed globally (in `/usr/local/bin/`) and for other cases, I use a global `.gitignore` file. It's fine, though.